modules/serverless-creating-kubeconfig-file.adoc

```diff
@@ -0,0 +1,18 @@
+// Module is included in the following assemblies:
+//
+// serverless/knative-client.adoc
+
+[id="create-kubeconfig-file_{contect}"]
+= Creating a `kubeconfig` file
+
+Use `kubeconfig` files to organize information about clusters, users, namespaces, and authentication mechanisms. The CLI tool uses `kubeconfig` files to communicate with the API server of a cluster.
+
+.Procedure
+* Create a basic `kubeconfig` file from client certificates. Use the following command:
+
+----
+$ oc adm create-kubeconfig \
+  --client-certificate=/path/to/client.crt \
+  --client-key=/path/to/client.key \
+  --certificate-authority=/path/to/ca.crt
```

Comment:
@navidshaikh is it enough to do it this way or do I have to also export the kubeconfig like this? `export KUBECONFIG=/path/tokubeconfig`

Reply:
@ mgencur I think it is mentioned the kubeconfig  install is optional.

Reply:
Should this be removed or made more explicit that it is optional?

Reply:
I think we should remove this and document `oc login` as the recommended way.

Reply:
@joaedwar can't find the place where it is mentioned it's optional. So maybe we could make it more explicit? thanks

Reply:
I'd say, since `oc login` is sufficient for `kn` to connect to cluster, we can remove this.